cmd/controller/main.go

```diff
@@ -89,10 +89,11 @@ func runController(watchNamespace string, imageServer imagehandler.ImageHandler,
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:    scheme,
-		Port:      0, // Add flag with default of 9443 when adding webhooks
-		Namespace: watchNamespace,
-		Cache:     cacheOptions,
+		Scheme:             scheme,
+		Port:               0, // Add flag with default of 9443 when adding webhooks
+		Namespace:          watchNamespace,
+		Cache:              cacheOptions,
+		MetricsBindAddress: metricsBindAddr,
```

Comment:
In general this lgtm, Just wondering if  we should add TLS Config like we have in BMO and CBO...